plugins/woocommerce-blocks/packages/checkout/filter-registry/test/admin.js

```diff
@@ -27,30 +24,35 @@ describe( 'Checkout registry (as admin user)', () => {
 			},
 		} );
 
-		const { result } = renderHook( () =>
+		try {
 			applyCheckoutFilter( {
 				filterName,
 				defaultValue: value,
-			} )
-		);
-		expect( result.error ).toEqual( Error( 'test error' ) );
+			} );
+		} catch ( e ) {
+			// eslint-disable-next-line  -- The toThrow helper does not stop wordpress/jest-console from erroring.
+			expect( e.message ).toBe( 'test error' );
+		}
 	} );
 
 	test( 'should throw if validation throws and user is an admin', () => {
+		expect.assertions( 1 );
```

Comment:
TIL

Reply:
Yeah me too, that was really interesting to read about! [Docs link](https://archive.jestjs.io/docs/en/22.x/expect#expectassertionsnumber) for folks who haven't seen this before.